openedx/features/badging/helpers.py

```diff
@@ -116,3 +120,32 @@ def filter_earned_badge_by_joined_team(user, course, earned_badges):
         earned_badge for earned_badge in earned_badges if
         earned_badge.community_id == course_team[TEAM_ROOM_ID_KEY]
     ]
+
+
+def send_user_badge_notification(user, my_badge_url, badge_name):
+    """
+    Send user new badge notification
+    :param user: Notification sender user
+    :param my_badge_url: Redirect url to my_badge view on notification click
+    :param badge_name: Newly earned badge
+    """
+    context = {
+        'badge_name': badge_name
+    }
+
+    body_short = render_to_string('philu_notifications/templates/user_badge_earned.html', context)
+
+    message_type = get_notification_type(
+        NOTIFICATION_TYPE.format(prefix='philu.badging', type='user-badge-earned')
+    )
+
+    message = NotificationMessage(
+        msg_type=message_type,
```

Comment:
This can be removed after adding EARNED_BADGE_NOTIFICATION
``` 
message_type = get_notification_type(
        NOTIFICATION_TYPE.format(prefix='philu.badging', type='user-badge-earned')
    )
```
and `msg_type=message_type,` can be updated to 
```msg_type= get_notification_type(EARNED_BADGE_NOTIFICATION),```

Reply:
removed